README.md

```diff
@@ -154,15 +193,15 @@ Write Ruby code that will create an instance of a person.
 Your answer...
 
 ```ruby
-Replace this with your answer
+wes= Person.new ("wes", 26)
```

Comment:
You need to indicate with attribute you're giving a value of "wes" and 26.

Also remember that `.new` doesn't save to the database. If you want to save, you'll need to follow `.new` up with a `.save`. You could also use `.create`, which will take care of both in one go.

So some possible answers...

```rb
# option A
wes = Person.new(name: "wes", age: 26)
wes.save

# option B
Person.create(name: "wes", age: 26)
```